torch/_inductor/select_algorithm.py

```diff
@@ -181,6 +184,26 @@ def stride(self, name, index):
             val = self.named_input_nodes[name].get_stride()[index]
         return texpr(self.rename_indexing(val))
 
+    def modification(self, fixed_inputs):
+        # HACK, but I can't figure out how to reuse existing Triton codegen properly
+        class PlaceholderSubstitution(V.MockHandler):
+            self.name = "PlaceholderSubstitution"
+
+            def load(self, name: str, index: sympy.Expr):
+                return f"({fixed_inputs[name]})"
+
+            def constant(self, value, dtype):
+                return TritonOverrides.constant(value, dtype)
+
+            def __getattr__(self, name):
+                return getattr(TritonOverrides, name)
+
+
+        with TritonKernel(1, 1, index_dtype=torch.int64) as kernel:
+            with V.set_ops_handler(PlaceholderSubstitution()):
+                out = self.subgraph.data.data.data.inner_fn((1,))
+                return out.value
```

Comment:
This will throw out the generated code, and only return a variable name.  To fetch the generated code you need to do something like:
https://github.com/pytorch/pytorch/blob/1aa92ae13df4a9c8094742f2d188e3b0caa26221/torch/_inductor/select_algorithm.py#L256-L257
after that you should run `self.body.clear()`.

You might need some of the indexing setup code in `store_output` as well if you want loading of unrelated buffers to work.  But it doesn't look like you do that in the current version.